examples/control access to spaces

```diff
@@ -0,0 +1,17 @@
+def update_space_access(space_id, group_id, permission_type)
+
+space = $sdk.space(space_id) 
+meta_id = space.content_metadata_id
+
+# Be sure the space isn't inheriting permissions. 
+$sdk.update_content_metadata(meta_id, {:inherits => false})
+
+all_accesss = $sdk.all_content_metadata_accesss({:content_metadata_id => meta_id}) 
```

Comment:
I think `all_content_metadata_accesss` is supposed to be `all_content_metadata_accesses`, it's missing an 'e' at the end.

Reply:
"Well, actually..."  it's misspelled in the metadata, byproduct of the pluralization pattern for the `get_all` endpoint type. I can fix it (breaking change) in API 3.1, but we have to leave API 3.0 as-is. :(

Reply:
Fixed in API 3.1 already.  3.0 is stuck with it.

Reply:
ahhh I guess I was looking at 3.1 calls in HT

Reply:
Might that be worth calling out in a comment then? I'm guessing this example could be used by people on 3.0 or 3.1 right?